### ISSUE TITLE
--colors=never still prints some colors

### ISSUE THREAD

**Issue by gerardroche** [laravel/framework#46759]
### Laravel Version

9.42.2

### PHP Version

8.2

### Description

In PHPUnit there is a `--colors` option:

```
--colors <flag>             Use colors in output ("never", "auto" or "always")
```

When running test via `artisan test --colors=never` stops printing *some* colors but not others. 

For example, this is the same or similar [issue that the Pest has](https://github.com/pestphp/pest/issues/778) i.e. it correctly disables some colors but not others like the test name:

**Colors enabled**

![Screenshot from 2023-04-13 04-10-36](https://user-images.githubusercontent.com/44148/231638676-a270b471-7a8e-4c39-85a3-00acf368502e.png)

**Colors disabled**

![Screenshot from 2023-04-13 04-10-48](https://user-images.githubusercontent.com/44148/231638682-fef2a819-0380-49a0-9963-1b54834d6033.png)

Notice the tests "one", "two", etc. are still output in color.

### Steps To Reproduce

Run `artisan test --colors=never` and observe color output.

**Comment by nunomaduro** [comment]
Lets discuss this on the collision repository.

**Comment by nunomaduro** [comment]
Can you create the issue here? https://github.com/nunomaduro/collision/issues/new.

**Comment by gerardroche** [comment]
@nunomaduro My apologies. I just updated to PHPUnit 10 and Pest 2.0, they updated the collision package and the issue seems to be resolved. Thanks.

**Comment by gerardroche** [comment]
@nunomaduro I got stuck trying to run the collision test suite. It kept failing and took me a while to figure out why. The first issue was I had pcov enabled and then I had cloned the repository to `~/vendor/nunomaduro/collission`. There is a if-else branch that tests if a file contains `/vendor/`, which of course every file contained :) See https://github.com/nunomaduro/collision/blob/v7.x/src/Writer.php#L286.
